printrun/printcore.py

```diff
@@ -416,7 +416,16 @@ def startprint(self, gcode, startindex = 0):
             return True
 
         self.clear = False
-        self._send("M110 N-1")
+        
+        '''Reset Gcode line number for printer, pls. see issue #1421.
```

Comment:
This information if very useful but I would not clutter the code with it. I believe this kind of explanation belongs to the commit message that introduces the code change, not in the code itself. If you know what I mean. Plus you wouldn't need to repeat it twice. I'm happy with the change otherwise.

Reply:
No problem, I removed the comments.